lib/scenic/adapters/postgres/refresh_dependencies.rb

```diff
@@ -0,0 +1,67 @@
+module Scenic
+  module Adapters
+    class Postgres
+      class RefreshDependencies
+        def self.call(name, adapter, connection)
+          new(name, adapter, connection).call
+        end
+
+        def initialize(name, adapter, connection)
+          @name = name
+          @adapter = adapter
+          @connection = connection
+        end
+
+        def call
+          dependencies.each do |dependency|
+            adapter.refresh_materialized_view(dependency)
+          end
+        end
+
+        private
+
+        attr_reader :name, :adapter, :connection
+
+        DEPENDENCY_SQL = <<-SQL.freeze
+          SELECT r_ns.nspname || '.' || cl_r.relname AS materialized_view,
+          array_agg(d_ns.nspname || '.' || cl_d.relname) AS depends_on
+          FROM pg_rewrite AS r
+          JOIN pg_class AS cl_r ON r.ev_class=cl_r.oid
```

Comment:
I'm having a hard time following the single letter and abreviated aliases here. Can we give them longer, more descriptive names to help maintenance going forward?